features/steps.rb

```diff
@@ -82,6 +82,15 @@
   run_silent %(git reset --quiet --hard HEAD^)
 end
 
+Given(/^the remote issues of "(.*?)" are:$/) do |proj, json_value|
```

Comment:
I don't feel there's a need for this custom step yet. How about simply using `Given the GitHub API server` directly?